src/renderer/components/ft-search-filters/ft-search-filters.vue

```diff
@@ -1,42 +1,44 @@
 <template>
   <div class="searchFilter">
```

Comment:
Remove the class from this `div` and from the CSS file, as it will no longer be used for anything.